full_hammerhead.mk

```diff
@@ -31,6 +31,8 @@ PRODUCT_MANUFACTURER := LGE
 PRODUCT_RESTRICT_VENDOR_FILES := true
 
 ENABLE_LIBRECOVERY := true
+RECOVERY_EXTERNAL_STORAGE := /data/media/0
+export USE_SET_METADATA := true
```

Comment:
nit: It might be worth adding a comment that this is done to enable the set_metadata() and set_metadata_recursive() functions in the generated updated script.